ccl/sqlccl: TestBackupAsOfSystemTime failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/eac297eeb9bb1f48fa558fad39d0032c8b59d1d7

Parameters:
```
COCKROACH_PROPOSER_EVALUATED_KV=true
TAGS=
GOFLAGS=
```

Stress build found a failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=203317&tab=buildLog

```
W170402 06:14:46.073580 22521 server/status/runtime.go:115  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170402 06:14:46.074663 22521 server/config.go:408  1 storage engine initialized
I170402 06:14:46.074885 22521 server/node.go:437  [n?] store [n0,s0] not bootstrapped
I170402 06:14:46.076488 22521 server/node.go:372  [n?] **** cluster b3da9bae-1bca-4b3e-91d3-e3f3c6b405f2 has been created
I170402 06:14:46.076503 22521 server/node.go:373  [n?] **** add additional nodes by specifying --join=127.0.0.1:39398
I170402 06:14:46.077692 22521 storage/store.go:1329  [n1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170402 06:14:46.077712 22521 server/node.go:450  [n1] initialized store [n1,s1]: {Capacity:536870912 Available:536870912 RangeCount:1 LeaseCount:0}
I170402 06:14:46.077728 22521 server/node.go:341  [n1] node ID 1 initialized
I170402 06:14:46.077753 22521 gossip/gossip.go:293  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:39398" > attrs:<> locality:<> 
I170402 06:14:46.077844 22521 storage/stores.go:296  [n1] read 0 node addresses from persistent storage
I170402 06:14:46.077881 22521 server/node.go:600  [n1] connecting to gossip network to verify cluster ID...
I170402 06:14:46.077935 22521 server/node.go:625  [n1] node connected via gossip and verified as part of cluster "b3da9bae-1bca-4b3e-91d3-e3f3c6b405f2"
I170402 06:14:46.077954 22521 server/node.go:388  [n1] node=1: started with [[]=] engine(s) and attributes []
I170402 06:14:46.078832 22677 storage/split_queue.go:98  [split,n1,s1,r1/1:/M{in-ax},@c42317c900] splitting at key /Table/0/0
I170402 06:14:46.083622 22677 storage/replica_command.go:2665  [split,n1,s1,r1/1:/M{in-ax},@c42317c900] initiating a split of this range at key /Table/0 [r2]
I170402 06:14:46.085100 22521 sql/executor.go:342  [n1] creating distSQLPlanner with address {tcp 127.0.0.1:39398}
E170402 06:14:46.090877 22678 storage/queue.go:634  [replicate,n1,s1,r1/1:/{Min-Table/0},@c42317c900] range requires a replication change, but lacks a quorum of live replicas (0/1)
I170402 06:14:46.091803 22521 server/server.go:695  [n1] starting https server at 127.0.0.1:42677
I170402 06:14:46.091820 22521 server/server.go:696  [n1] starting grpc/postgres server at 127.0.0.1:39398
I170402 06:14:46.091827 22521 server/server.go:697  [n1] advertising CockroachDB node at 127.0.0.1:39398
I170402 06:14:46.094141 22677 storage/split_queue.go:98  [split,n1,s1,r2/1:/{Table/0-Max},@c423cd8900] splitting at key /Table/11/0
I170402 06:14:46.094172 22677 storage/replica_command.go:2665  [split,n1,s1,r2/1:/{Table/0-Max},@c423cd8900] initiating a split of this range at key /Table/11 [r3]
I170402 06:14:46.099095 22677 storage/split_queue.go:98  [split,n1,s1,r3/1:/{Table/11-Max},@c42317c600] splitting at key /Table/12/0
I170402 06:14:46.099125 22677 storage/replica_command.go:2665  [split,n1,s1,r3/1:/{Table/11-Max},@c42317c600] initiating a split of this range at key /Table/12 [r4]
I170402 06:14:46.101720 22521 sql/event_log.go:101  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN uniqueID SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I170402 06:14:46.103664 22677 storage/split_queue.go:98  [split,n1,s1,r4/1:/{Table/12-Max},@c4234ebb00] splitting at key /Table/13/0
I170402 06:14:46.103695 22677 storage/replica_command.go:2665  [split,n1,s1,r4/1:/{Table/12-Max},@c4234ebb00] initiating a split of this range at key /Table/13 [r5]
I170402 06:14:46.104979 22521 sql/lease.go:372  [n1] publish: descID=12 (eventlog) version=2 mtime=2017-04-02 06:14:46.104962087 +0000 UTC
I170402 06:14:46.108167 22677 storage/split_queue.go:98  [split,n1,s1,r5/1:/{Table/13-Max},@c421f7af00] splitting at key /Table/14/0
I170402 06:14:46.108203 22677 storage/replica_command.go:2665  [split,n1,s1,r5/1:/{Table/13-Max},@c421f7af00] initiating a split of this range at key /Table/14 [r6]
I170402 06:14:46.110693 22521 server/server.go:805  [n1] done ensuring all necessary migrations have run
I170402 06:14:46.110713 22521 server/server.go:807  [n1] serving sql connections
I170402 06:14:46.118057 22677 storage/split_queue.go:98  [split,n1,s1,r6/1:/{Table/14-Max},@c42073c900] splitting at key /Table/15/0
I170402 06:14:46.118089 22677 storage/replica_command.go:2665  [split,n1,s1,r6/1:/{Table/14-Max},@c42073c900] initiating a split of this range at key /Table/15 [r7]
I170402 06:14:46.119615 22703 sql/event_log.go:101  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:39398} Attrs: Locality:} ClusterID:b3da9bae-1bca-4b3e-91d3-e3f3c6b405f2 StartedAt:1491113686077943049 LastUp:1491113686077943049}
W170402 06:14:46.126750 22521 gossip/gossip.go:1178  [n?] no incoming or outgoing connections
W170402 06:14:46.136591 22521 server/status/runtime.go:115  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170402 06:14:46.140509 22521 server/config.go:408  1 storage engine initialized
I170402 06:14:46.140748 22521 server/node.go:437  [n?] store [n0,s0] not bootstrapped
I170402 06:14:46.140767 22521 storage/stores.go:296  [n?] read 0 node addresses from persistent storage
I170402 06:14:46.140784 22521 server/node.go:600  [n?] connecting to gossip network to verify cluster ID...
I170402 06:14:46.145920 23032 gossip/client.go:131  [n?] started gossip client to 127.0.0.1:39398
I170402 06:14:46.146136 22986 gossip/server.go:234  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:56984}
I170402 06:14:46.147272 23058 storage/stores.go:312  [n?] wrote 1 node addresses to persistent storage
I170402 06:14:46.147348 22521 server/node.go:625  [n?] node connected via gossip and verified as part of cluster "b3da9bae-1bca-4b3e-91d3-e3f3c6b405f2"
I170402 06:14:46.148989 22521 kv/dist_sender.go:368  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170402 06:14:46.149773 22521 server/node.go:334  [n?] new node allocated ID 2
I170402 06:14:46.149814 22521 gossip/gossip.go:293  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:56984" > attrs:<> locality:<> 
I170402 06:14:46.149861 22521 server/node.go:388  [n2] node=2: started with [[]=] engine(s) and attributes []
I170402 06:14:46.149881 22521 sql/executor.go:342  [n2] creating distSQLPlanner with address {tcp 127.0.0.1:56984}
I170402 06:14:46.150490 23055 storage/stores.go:312  [n1] wrote 1 node addresses to persistent storage
I170402 06:14:46.151669 22521 server/server.go:695  [n2] starting https server at 127.0.0.1:47290
I170402 06:14:46.151686 22521 server/server.go:696  [n2] starting grpc/postgres server at 127.0.0.1:56984
I170402 06:14:46.151693 22521 server/server.go:697  [n2] advertising CockroachDB node at 127.0.0.1:56984
I170402 06:14:46.152729 23066 server/node.go:581  [n2] bootstrapped store [n2,s2]
I170402 06:14:46.153142 22850 storage/replica_raftstorage.go:413  [replicate,n1,s1,r4/1:/Table/1{2-3},@c4234ebb00] generated preemptive snapshot b4c06e05 at index 18
I170402 06:14:46.154489 22521 server/server.go:805  [n2] done ensuring all necessary migrations have run
I170402 06:14:46.154526 22521 server/server.go:807  [n2] serving sql connections
I170402 06:14:46.160358 23143 sql/event_log.go:101  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:56984} Attrs: Locality:} ClusterID:b3da9bae-1bca-4b3e-91d3-e3f3c6b405f2 StartedAt:1491113686149851007 LastUp:1491113686149851007}
W170402 06:14:46.161129 22521 gossip/gossip.go:1178  [n?] no incoming or outgoing connections
W170402 06:14:46.164789 22521 server/status/runtime.go:115  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170402 06:14:46.165963 22521 server/config.go:408  1 storage engine initialized
I170402 06:14:46.166197 22521 server/node.go:437  [n?] store [n0,s0] not bootstrapped
I170402 06:14:46.166212 22521 storage/stores.go:296  [n?] read 0 node addresses from persistent storage
I170402 06:14:46.166225 22850 storage/store.go:3356  [replicate,n1,s1,r4/1:/Table/1{2-3},@c4234ebb00] streamed snapshot: kv pairs: 20, log entries: 8, 0ms
I170402 06:14:46.166230 22521 server/node.go:600  [n?] connecting to gossip network to verify cluster ID...
I170402 06:14:46.166475 22651 storage/replica_raftstorage.go:593  [n2,s2,r4/?:{-},@c421f7a000] applying preemptive snapshot at index 18 (id=b4c06e05, encoded size=12018, 1 rocksdb batches, 8 log entries)
I170402 06:14:46.166720 22651 storage/replica_raftstorage.go:601  [n2,s2,r4/?:/Table/1{2-3},@c421f7a000] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.167112 22850 storage/replica_command.go:3560  [replicate,n1,s1,r4/1:/Table/1{2-3},@c4234ebb00] change replicas (ADD_REPLICA {2 2 2}): read existing descriptor range_id:4 start_key:"\224" end_key:"\225" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170402 06:14:46.169633 23221 storage/replica.go:2628  [n1,s1,r4/1:/Table/1{2-3},@c4234ebb00] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170402 06:14:46.170109 22850 storage/replica_raftstorage.go:413  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421f7af00] generated preemptive snapshot 5d6c47aa at index 24
I170402 06:14:46.170613 22850 storage/store.go:3356  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421f7af00] streamed snapshot: kv pairs: 56, log entries: 14, 0ms
I170402 06:14:46.170924 23234 storage/replica_raftstorage.go:593  [n2,s2,r5/?:{-},@c423cd8000] applying preemptive snapshot at index 24 (id=5d6c47aa, encoded size=25967, 1 rocksdb batches, 14 log entries)
I170402 06:14:46.171230 23234 storage/replica_raftstorage.go:601  [n2,s2,r5/?:/Table/1{3-4},@c423cd8000] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.171305 23251 storage/raft_transport.go:436  [n2] raft transport stream to node 1 established
I170402 06:14:46.171680 22850 storage/replica_command.go:3560  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421f7af00] change replicas (ADD_REPLICA {2 2 2}): read existing descriptor range_id:5 start_key:"\225" end_key:"\226" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170402 06:14:46.174539 23262 storage/replica.go:2628  [n1,s1,r5/1:/Table/1{3-4},@c421f7af00] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170402 06:14:46.175006 22850 storage/replica_raftstorage.go:413  [replicate,n1,s1,r6/1:/Table/1{4-5},@c42073c900] generated preemptive snapshot f3b14437 at index 18
I170402 06:14:46.175397 22850 storage/store.go:3356  [replicate,n1,s1,r6/1:/Table/1{4-5},@c42073c900] streamed snapshot: kv pairs: 10, log entries: 8, 0ms
I170402 06:14:46.175679 23266 storage/replica_raftstorage.go:593  [n2,s2,r6/?:{-},@c421f7b800] applying preemptive snapshot at index 18 (id=f3b14437, encoded size=7843, 1 rocksdb batches, 8 log entries)
I170402 06:14:46.175702 22671 gossip/client.go:131  [n?] started gossip client to 127.0.0.1:39398
I170402 06:14:46.175890 23266 storage/replica_raftstorage.go:601  [n2,s2,r6/?:/Table/1{4-5},@c421f7b800] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.175997 23182 gossip/server.go:234  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:33071}
I170402 06:14:46.176149 22521 server/node.go:625  [n?] node connected via gossip and verified as part of cluster "b3da9bae-1bca-4b3e-91d3-e3f3c6b405f2"
I170402 06:14:46.176212 23284 storage/stores.go:312  [n?] wrote 1 node addresses to persistent storage
I170402 06:14:46.176217 22850 storage/replica_command.go:3560  [replicate,n1,s1,r6/1:/Table/1{4-5},@c42073c900] change replicas (ADD_REPLICA {2 2 2}): read existing descriptor range_id:6 start_key:"\226" end_key:"\227" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170402 06:14:46.176266 23284 storage/stores.go:312  [n?] wrote 2 node addresses to persistent storage
I170402 06:14:46.177139 22521 kv/dist_sender.go:368  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170402 06:14:46.177610 22521 server/node.go:334  [n?] new node allocated ID 3
I170402 06:14:46.177648 22521 gossip/gossip.go:293  [n3] NodeDescriptor set to node_id:3 address:<network_field:"tcp" address_field:"127.0.0.1:33071" > attrs:<> locality:<> 
I170402 06:14:46.177686 22521 server/node.go:388  [n3] node=3: started with [[]=] engine(s) and attributes []
I170402 06:14:46.177706 22521 sql/executor.go:342  [n3] creating distSQLPlanner with address {tcp 127.0.0.1:33071}
I170402 06:14:46.178404 23274 storage/stores.go:312  [n1] wrote 2 node addresses to persistent storage
I170402 06:14:46.178586 23275 storage/stores.go:312  [n2] wrote 2 node addresses to persistent storage
I170402 06:14:46.179405 22521 server/server.go:695  [n3] starting https server at 127.0.0.1:52106
I170402 06:14:46.179422 22521 server/server.go:696  [n3] starting grpc/postgres server at 127.0.0.1:33071
I170402 06:14:46.179429 22521 server/server.go:697  [n3] advertising CockroachDB node at 127.0.0.1:33071
I170402 06:14:46.179439 23347 storage/replica.go:2628  [n1,s1,r6/1:/Table/1{4-5},@c42073c900] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170402 06:14:46.179874 22850 storage/replica_raftstorage.go:413  [replicate,n1,s1,r7/1:/{Table/15-Max},@c421207500] generated preemptive snapshot 58c78e7e at index 11
I170402 06:14:46.180223 22850 storage/store.go:3356  [replicate,n1,s1,r7/1:/{Table/15-Max},@c421207500] streamed snapshot: kv pairs: 9, log entries: 1, 0ms
I170402 06:14:46.180474 23160 storage/replica_raftstorage.go:593  [n2,s2,r7/?:{-},@c4234ea300] applying preemptive snapshot at index 11 (id=58c78e7e, encoded size=508, 1 rocksdb batches, 1 log entries)
I170402 06:14:46.180644 23160 storage/replica_raftstorage.go:601  [n2,s2,r7/?:/{Table/15-Max},@c4234ea300] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.180787 22521 server/server.go:805  [n3] done ensuring all necessary migrations have run
I170402 06:14:46.180808 22521 server/server.go:807  [n3] serving sql connections
I170402 06:14:46.181426 22850 storage/replica_command.go:3560  [replicate,n1,s1,r7/1:/{Table/15-Max},@c421207500] change replicas (ADD_REPLICA {2 2 2}): read existing descriptor range_id:7 start_key:"\227" end_key:"\377\377" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170402 06:14:46.181760 23301 server/node.go:581  [n3] bootstrapped store [n3,s3]
I170402 06:14:46.188459 22850 storage/replica_command.go:3560  [replicate,n1,s1,r7/1:/{Table/15-Max},@c421207500] change replicas (ADD_REPLICA {2 2 2}): read existing descriptor range_id:7 start_key:"\227" end_key:"\377\377" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170402 06:14:46.191849 23423 storage/replica.go:2628  [n1,s1,r7/1:/{Table/15-Max},@c421207500] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170402 06:14:46.192324 22850 storage/replica_raftstorage.go:413  [replicate,n1,s1,r2/1:/Table/{0-11},@c423cd8900] generated preemptive snapshot a7ffa9d5 at index 26
I170402 06:14:46.192474 23278 sql/event_log.go:101  [n3] Event: "node_join", target: 3, info: {Descriptor:{NodeID:3 Address:{NetworkField:tcp AddressField:127.0.0.1:33071} Attrs: Locality:} ClusterID:b3da9bae-1bca-4b3e-91d3-e3f3c6b405f2 StartedAt:1491113686177676303 LastUp:1491113686177676303}
I170402 06:14:46.202284 22850 storage/store.go:3356  [replicate,n1,s1,r2/1:/Table/{0-11},@c423cd8900] streamed snapshot: kv pairs: 35, log entries: 16, 0ms
I170402 06:14:46.202533 23383 storage/replica_raftstorage.go:593  [n3,s3,r2/?:{-},@c421f7b200] applying preemptive snapshot at index 26 (id=a7ffa9d5, encoded size=25554, 1 rocksdb batches, 16 log entries)
I170402 06:14:46.202858 23383 storage/replica_raftstorage.go:601  [n3,s3,r2/?:/Table/{0-11},@c421f7b200] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.203195 22850 storage/replica_command.go:3560  [replicate,n1,s1,r2/1:/Table/{0-11},@c423cd8900] change replicas (ADD_REPLICA {3 3 2}): read existing descriptor range_id:2 start_key:"\210" end_key:"\223" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170402 06:14:46.205931 23454 storage/replica.go:2628  [n1,s1,r2/1:/Table/{0-11},@c423cd8900] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2}]
I170402 06:14:46.206388 22850 storage/replica_raftstorage.go:413  [replicate,n1,s1,r3/1:/Table/1{1-2},@c42317c600] generated preemptive snapshot e4daf54c at index 18
I170402 06:14:46.206782 22850 storage/store.go:3356  [replicate,n1,s1,r3/1:/Table/1{1-2},@c42317c600] streamed snapshot: kv pairs: 10, log entries: 8, 0ms
I170402 06:14:46.207099 23491 storage/replica_raftstorage.go:593  [n3,s3,r3/?:{-},@c423cd8c00] applying preemptive snapshot at index 18 (id=e4daf54c, encoded size=10500, 1 rocksdb batches, 8 log entries)
I170402 06:14:46.207379 23491 storage/replica_raftstorage.go:601  [n3,s3,r3/?:/Table/1{1-2},@c423cd8c00] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.207743 22850 storage/replica_command.go:3560  [replicate,n1,s1,r3/1:/Table/1{1-2},@c42317c600] change replicas (ADD_REPLICA {3 3 2}): read existing descriptor range_id:3 start_key:"\223" end_key:"\224" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170402 06:14:46.207760 23506 storage/raft_transport.go:436  [n3] raft transport stream to node 1 established
I170402 06:14:46.210661 23401 storage/replica.go:2628  [n1,s1,r3/1:/Table/1{1-2},@c42317c600] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2}]
I170402 06:14:46.210991 22850 storage/queue.go:699  [n1,replicate] purgatory is now empty
I170402 06:14:46.211201 22678 storage/replica_raftstorage.go:413  [replicate,n1,s1,r4/1:/Table/1{2-3},@c4234ebb00] generated preemptive snapshot debdf9e3 at index 25
I170402 06:14:46.211692 22678 storage/store.go:3356  [replicate,n1,s1,r4/1:/Table/1{2-3},@c4234ebb00] streamed snapshot: kv pairs: 31, log entries: 15, 0ms
I170402 06:14:46.211923 23493 storage/replica_raftstorage.go:593  [n3,s3,r4/?:{-},@c422548300] applying preemptive snapshot at index 25 (id=debdf9e3, encoded size=23648, 1 rocksdb batches, 15 log entries)
I170402 06:14:46.212241 23493 storage/replica_raftstorage.go:601  [n3,s3,r4/?:/Table/1{2-3},@c422548300] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.212619 22678 storage/replica_command.go:3560  [replicate,n1,s1,r4/1:/Table/1{2-3},@c4234ebb00] change replicas (ADD_REPLICA {3 3 3}): read existing descriptor range_id:4 start_key:"\224" end_key:"\225" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170402 06:14:46.215572 23387 storage/replica.go:2628  [n1,s1,r4/1:/Table/1{2-3},@c4234ebb00] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170402 06:14:46.216688 22678 storage/replica_raftstorage.go:413  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421f7af00] generated preemptive snapshot 0d2c10c9 at index 39
I170402 06:14:46.217697 22678 storage/store.go:3356  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421f7af00] streamed snapshot: kv pairs: 87, log entries: 29, 0ms
I170402 06:14:46.218053 23407 storage/replica_raftstorage.go:593  [n3,s3,r5/?:{-},@c422548c00] applying preemptive snapshot at index 39 (id=0d2c10c9, encoded size=52598, 1 rocksdb batches, 29 log entries)
I170402 06:14:46.218553 23407 storage/replica_raftstorage.go:601  [n3,s3,r5/?:/Table/1{3-4},@c422548c00] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.219105 22678 storage/replica_command.go:3560  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421f7af00] change replicas (ADD_REPLICA {3 3 3}): read existing descriptor range_id:5 start_key:"\225" end_key:"\226" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170402 06:14:46.222954 23554 storage/replica.go:2628  [n1,s1,r5/1:/Table/1{3-4},@c421f7af00] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170402 06:14:46.223815 22678 storage/replica_raftstorage.go:413  [replicate,n1,s1,r3/1:/Table/1{1-2},@c42317c600] generated preemptive snapshot 485172f4 at index 21
I170402 06:14:46.224281 22678 storage/store.go:3356  [replicate,n1,s1,r3/1:/Table/1{1-2},@c42317c600] streamed snapshot: kv pairs: 11, log entries: 11, 0ms
I170402 06:14:46.224553 23557 storage/replica_raftstorage.go:593  [n2,s2,r3/?:{-},@c421f60c00] applying preemptive snapshot at index 21 (id=485172f4, encoded size=13470, 1 rocksdb batches, 11 log entries)
I170402 06:14:46.224796 23557 storage/replica_raftstorage.go:601  [n2,s2,r3/?:/Table/1{1-2},@c421f60c00] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.225515 22678 storage/replica_command.go:3560  [replicate,n1,s1,r3/1:/Table/1{1-2},@c42317c600] change replicas (ADD_REPLICA {2 2 3}): read existing descriptor range_id:3 start_key:"\223" end_key:"\224" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:3 store_id:3 replica_id:2 > next_replica_id:3 
I170402 06:14:46.228728 23535 storage/replica.go:2628  [n1,s1,r3/1:/Table/1{1-2},@c42317c600] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2} {NodeID:2 StoreID:2 ReplicaID:3}]
I170402 06:14:46.229611 22678 storage/replica_raftstorage.go:413  [replicate,n1,s1,r2/1:/Table/{0-11},@c423cd8900] generated preemptive snapshot 47f2e7c7 at index 29
I170402 06:14:46.230202 22678 storage/store.go:3356  [replicate,n1,s1,r2/1:/Table/{0-11},@c423cd8900] streamed snapshot: kv pairs: 36, log entries: 19, 0ms
I170402 06:14:46.230507 23465 storage/replica_raftstorage.go:593  [n2,s2,r2/?:{-},@c4239d2900] applying preemptive snapshot at index 29 (id=47f2e7c7, encoded size=28524, 1 rocksdb batches, 19 log entries)
I170402 06:14:46.230861 23465 storage/replica_raftstorage.go:601  [n2,s2,r2/?:/Table/{0-11},@c4239d2900] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.231270 22678 storage/replica_command.go:3560  [replicate,n1,s1,r2/1:/Table/{0-11},@c423cd8900] change replicas (ADD_REPLICA {2 2 3}): read existing descriptor range_id:2 start_key:"\210" end_key:"\223" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:3 store_id:3 replica_id:2 > next_replica_id:3 
I170402 06:14:46.234353 23572 storage/replica.go:2628  [n1,s1,r2/1:/Table/{0-11},@c423cd8900] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2} {NodeID:2 StoreID:2 ReplicaID:3}]
I170402 06:14:46.235199 22678 storage/replica_raftstorage.go:413  [replicate,n1,s1,r7/1:/{Table/15-Max},@c421207500] generated preemptive snapshot d98a950f at index 16
I170402 06:14:46.235545 22678 storage/store.go:3356  [replicate,n1,s1,r7/1:/{Table/15-Max},@c421207500] streamed snapshot: kv pairs: 10, log entries: 6, 0ms
I170402 06:14:46.235805 23634 storage/replica_raftstorage.go:593  [n3,s3,r7/?:{-},@c421369800] applying preemptive snapshot at index 16 (id=d98a950f, encoded size=5462, 1 rocksdb batches, 6 log entries)
I170402 06:14:46.236020 23634 storage/replica_raftstorage.go:601  [n3,s3,r7/?:/{Table/15-Max},@c421369800] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.236452 22678 storage/replica_command.go:3560  [replicate,n1,s1,r7/1:/{Table/15-Max},@c421207500] change replicas (ADD_REPLICA {3 3 3}): read existing descriptor range_id:7 start_key:"\227" end_key:"\377\377" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170402 06:14:46.239230 23621 storage/replica.go:2628  [n1,s1,r7/1:/{Table/15-Max},@c421207500] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170402 06:14:46.239992 22678 storage/replica_raftstorage.go:413  [replicate,n1,s1,r6/1:/Table/1{4-5},@c42073c900] generated preemptive snapshot 492eb1ce at index 21
I170402 06:14:46.240402 22678 storage/store.go:3356  [replicate,n1,s1,r6/1:/Table/1{4-5},@c42073c900] streamed snapshot: kv pairs: 11, log entries: 11, 0ms
I170402 06:14:46.240815 23580 storage/replica_raftstorage.go:593  [n3,s3,r6/?:{-},@c421369b00] applying preemptive snapshot at index 21 (id=492eb1ce, encoded size=10813, 1 rocksdb batches, 11 log entries)
I170402 06:14:46.241059 23580 storage/replica_raftstorage.go:601  [n3,s3,r6/?:/Table/1{4-5},@c421369b00] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170402 06:14:46.241412 22678 storage/replica_command.go:3560  [replicate,n1,s1,r6/1:/Table/1{4-5},@c42073c900] change replicas (ADD_REPLICA {3 3 3}): read existing descriptor range_id:6 start_key:"\226" end_key:"\227" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170402 06:14:46.244453 23652 storage/replica.go:2628  [n1,s1,r6/1:/Table/1{4-5},@c42073c900] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170402 06:14:46.278315 22678 storage/replica_raftstorage.go:413  [replicate,n1,s1,r1/1:/{Min-Table/0},@c42317c900] generated preemptive snapshot d536f833 at index 80
I170402 06:14:46.282971 22678 storage/store.go:3356  [replicate,n1,s1,r1/1:/{Min-Table/0},@c42317c900] streamed snapshot: kv pairs: 841, log entries: 70, 4ms
I170402 06:14:46.284544 23627 storage/replica_raftstorage.go:593  [n3,s3,r1/?:{-},@c423680c00] applying preemptive snapshot at index 80 (id=d536f833, encoded size=447223, 1 rocksdb batches, 70 log entries)
I170402 06:14:46.288718 23627 storage/replica_raftstorage.go:601  [n3,s3,r1/?:/{Min-Table/0},@c423680c00] applied preemptive snapshot in 4ms [clear=0ms batch=0ms entries=2ms commit=1ms]
I170402 06:14:46.289540 22678 storage/replica_command.go:3560  [replicate,n1,s1,r1/1:/{Min-Table/0},@c42317c900] change replicas (ADD_REPLICA {3 3 2}): read existing descriptor range_id:1 start_key:"" end_key:"\210" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170402 06:14:46.293587 23628 storage/replica.go:2628  [n1,s1,r1/1:/{Min-Table/0},@c42317c900] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2}]
I170402 06:14:46.294337 22678 storage/replica_raftstorage.go:413  [replicate,n1,s1,r1/1:/{Min-Table/0},@c42317c900] generated preemptive snapshot f9847c2c at index 85
I170402 06:14:46.299039 22678 storage/store.go:3356  [replicate,n1,s1,r1/1:/{Min-Table/0},@c42317c900] streamed snapshot: kv pairs: 847, log entries: 75, 5ms
I170402 06:14:46.299433 23585 storage/replica_raftstorage.go:593  [n2,s2,r1/?:{-},@c421207b00] applying preemptive snapshot at index 85 (id=f9847c2c, encoded size=452201, 1 rocksdb batches, 75 log entries)
I170402 06:14:46.301455 23585 storage/replica_raftstorage.go:601  [n2,s2,r1/?:/{Min-Table/0},@c421207b00] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170402 06:14:46.301823 22678 storage/replica_command.go:3560  [replicate,n1,s1,r1/1:/{Min-Table/0},@c42317c900] change replicas (ADD_REPLICA {2 2 3}): read existing descriptor range_id:1 start_key:"" end_key:"\210" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:3 store_id:3 replica_id:2 > next_replica_id:3 
I170402 06:14:46.305802 23594 storage/replica.go:2628  [n1,s1,r1/1:/{Min-Table/0},@c42317c900] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2} {NodeID:2 StoreID:2 ReplicaID:3}]
I170402 06:14:46.441510 23022 sql/event_log.go:101  [client=127.0.0.1:43478,user=root,n1] Event: "create_database", target: 50, info: {DatabaseName:bench Statement:CREATE DATABASE bench User:root}
I170402 06:14:46.442393 22677 storage/split_queue.go:98  [split,n1,s1,r7/1:/{Table/15-Max},@c421207500] splitting at key /Table/50/0
I170402 06:14:46.442426 22677 storage/replica_command.go:2665  [split,n1,s1,r7/1:/{Table/15-Max},@c421207500] initiating a split of this range at key /Table/50 [r8]
I170402 06:14:46.449934 23022 sql/event_log.go:101  [client=127.0.0.1:43478,user=root,n1] Event: "create_table", target: 51, info: {TableName:bench.bank Statement:CREATE TABLE bench.bank (id INT PRIMARY KEY, balance INT, payload STRING, FAMILY (id, balance, payload)) User:root}
I170402 06:14:46.450427 22677 storage/split_queue.go:98  [split,n1,s1,r8/1:/{Table/50-Max},@c4214f8600] splitting at key /Table/51/0
I170402 06:14:46.450457 22677 storage/replica_command.go:2665  [split,n1,s1,r8/1:/{Table/50-Max},@c4214f8600] initiating a split of this range at key /Table/51 [r9]
I170402 06:14:46.497520 23022 sql/mon/mem_usage.go:509  [client=127.0.0.1:43478,user=root,n1] txn: memory usage increases to 49 KiB (+49680)
I170402 06:14:46.497579 23022 sql/mon/mem_usage.go:509  [client=127.0.0.1:43478,user=root,n1] txn: memory usage increases to 64 KiB (+140)
I170402 06:14:46.497698 23022 sql/mon/mem_usage.go:509  [client=127.0.0.1:43478,user=root,n1] txn: memory usage increases to 128 KiB (+140)
I170402 06:14:46.556328 23769 storage/replica_command.go:2665  [n1,s1,r9/1:/{Table/51-Max},@c42273c600] initiating a split of this range at key /Table/51/1/100 [r10]
I170402 06:14:46.568443 23847 storage/replica_command.go:2665  [n1,s1,r10/1:/{Table/51/1/1…-Max},@c421368600] initiating a split of this range at key /Table/51/1/200 [r11]
I170402 06:14:46.578120 23912 storage/replica_command.go:2665  [n1,s1,r11/1:/{Table/51/1/2…-Max},@c42140ac00] initiating a split of this range at key /Table/51/1/300 [r12]
I170402 06:14:46.584507 23942 storage/replica_command.go:2665  [n1,s1,r12/1:/{Table/51/1/3…-Max},@c423680000] initiating a split of this range at key /Table/51/1/400 [r13]
I170402 06:14:46.593322 23809 storage/replica_command.go:2665  [n1,s1,r13/1:/{Table/51/1/4…-Max},@c4210ed200] initiating a split of this range at key /Table/51/1/500 [r14]
I170402 06:14:46.603083 23974 storage/replica_command.go:2665  [n1,s1,r14/1:/{Table/51/1/5…-Max},@c42073cf00] initiating a split of this range at key /Table/51/1/600 [r15]
I170402 06:14:46.625923 24069 storage/replica_command.go:2665  [n1,s1,r15/1:/{Table/51/1/6…-Max},@c4238de000] initiating a split of this range at key /Table/51/1/700 [r16]
I170402 06:14:46.637334 23998 storage/replica_command.go:2665  [n1,s1,r16/1:/{Table/51/1/7…-Max},@c421ae0300] initiating a split of this range at key /Table/51/1/800 [r17]
I170402 06:14:49.886654 24210 storage/replica_command.go:2665  [n1,s1,r17/1:/{Table/51/1/8…-Max},@c4239d3200] initiating a split of this range at key /Table/51/1/900 [r18]
I170402 06:14:49.971987 24313 ccl/storageccl/export.go:68  [n1,s1,r18/1:/{Table/51/1/9…-Max},@c4234c4c00] export [/Table/51/1/900,/Table/51/2)
I170402 06:14:49.972803 24300 ccl/storageccl/export.go:68  [n1,s1,r10/1:/Table/51/1/{100-200},@c421368600] export [/Table/51/1/100,/Table/51/1/200)
I170402 06:14:49.974825 24177 ccl/storageccl/export.go:68  [n1,s1,r14/1:/Table/51/1/{500-600},@c42073cf00] export [/Table/51/1/500,/Table/51/1/600)
I170402 06:14:49.975055 24122 ccl/storageccl/export.go:68  [n1,s1,r9/1:/Table/51{-/1/100},@c42273c600] export [/Table/51/1,/Table/51/1/100)
I170402 06:14:49.975096 24247 ccl/storageccl/export.go:68  [n1,s1,r2/1:/Table/{0-11},@c423cd8900] export [/Table/3/1,/Table/3/2)
I170402 06:14:49.992793 24225 ccl/storageccl/export.go:68  [n1,s1,r11/1:/Table/51/1/{200-300},@c42140ac00] export [/Table/51/1/200,/Table/51/1/300)
I170402 06:14:49.992944 24224 ccl/storageccl/export.go:68  [n1,s1,r12/1:/Table/51/1/{300-400},@c423680000] export [/Table/51/1/300,/Table/51/1/400)
I170402 06:14:49.994863 24322 ccl/storageccl/export.go:68  [n1,s1,r2/1:/Table/{0-11},@c423cd8900] export [/Table/4/1,/Table/4/2)
I170402 06:14:49.995094 24323 ccl/storageccl/export.go:68  [n1,s1,r13/1:/Table/51/1/{400-500},@c4210ed200] export [/Table/51/1/400,/Table/51/1/500)
I170402 06:14:49.995856 24324 ccl/storageccl/export.go:68  [n1,s1,r16/1:/Table/51/1/{700-800},@c421ae0300] export [/Table/51/1/700,/Table/51/1/800)
I170402 06:14:50.008130 24325 ccl/storageccl/export.go:68  [n1,s1,r15/1:/Table/51/1/{600-700},@c4238de000] export [/Table/51/1/600,/Table/51/1/700)
I170402 06:14:50.026171 24326 ccl/storageccl/export.go:68  [n1,s1,r17/1:/Table/51/1/{800-900},@c4239d3200] export [/Table/51/1/800,/Table/51/1/900)
I170402 06:14:50.102928 23022 sql/event_log.go:101  [client=127.0.0.1:43478,user=root,n1] Event: "drop_table", target: 51, info: {TableName:bank Statement:DROP TABLE bench.bank User:root CascadeDroppedViews:[]}
I170402 06:14:50.108697 23022 sql/lease.go:372  [client=127.0.0.1:43478,user=root,n1] publish: descID=51 (bank) version=2 mtime=2017-04-02 06:14:50.108681322 +0000 UTC
I170402 06:14:50.112299 23022 sql/lease.go:301  publish (count leases): descID=51 name=bank version=1 count=1
I170402 06:14:50.183816 24317 storage/replica_command.go:2665  [n1,s1,r18/1:/{Table/51/1/9…-Max},@c4234c4c00] initiating a split of this range at key /Table/52/1/500 [r19]
I170402 06:14:50.208823 24536 storage/replica_command.go:2665  [n1,s1,r18/1:/Table/5{1/1/900-2/1/500},@c4234c4c00] initiating a split of this range at key /Table/52/1/200 [r20]
I170402 06:14:50.208913 24535 storage/replica_command.go:2665  [n1,s1,r19/1:/{Table/52/1/5…-Max},@c421536600] initiating a split of this range at key /Table/52/1/800 [r21]
I170402 06:14:50.216768 24568 storage/replica_command.go:2665  [n1,s1,r20/1:/Table/52/1/{200-500},@c420967b00] initiating a split of this range at key /Table/52/1/400 [r22]
I170402 06:14:50.217711 24575 storage/replica_command.go:2665  [n1,s1,r18/1:/Table/5{1/1/900-2/1/200},@c4234c4c00] initiating a split of this range at key /Table/52/1/100 [r23]
I170402 06:14:50.231690 24613 storage/replica_command.go:2665  [n1,s1,r21/1:/{Table/52/1/8…-Max},@c42387b200] initiating a split of this range at key /Table/52/1/900 [r24]
I170402 06:14:50.233244 24621 storage/replica_command.go:2665  [n1,s1,r19/1:/Table/52/1/{500-800},@c421536600] initiating a split of this range at key /Table/52/1/700 [r25]
I170402 06:14:50.241527 24677 storage/replica_command.go:2665  [n1,s1,r18/1:/Table/5{1/1/900-2/1/100},@c4234c4c00] initiating a split of this range at key /Table/52/1 [r26]
I170402 06:14:50.254521 24605 storage/replica_command.go:2665  [n1,s1,r19/1:/Table/52/1/{500-700},@c421536600] initiating a split of this range at key /Table/52/1/600 [r27]
I170402 06:14:53.153662 24792 storage/raft_transport.go:436  [n2] raft transport stream to node 3 established
I170402 06:14:53.154220 24797 storage/raft_transport.go:436  [n3] raft transport stream to node 2 established
I170402 06:14:54.960562 24868 storage/replica_command.go:2665  [n1,s1,r20/1:/Table/52/1/{200-400},@c420967b00] initiating a split of this range at key /Table/52/1/300 [r28]
I170402 06:14:54.969419 24932 ccl/storageccl/import.go:57  [n1,s1,r20/1:/Table/52/1/{200-300},@c420967b00] import [/Table/52/1/200,/Table/52/1/300)
I170402 06:14:54.969536 24878 ccl/storageccl/import.go:57  [n1,s1,r21/1:/Table/52/1/{800-900},@c42387b200] import [/Table/52/1/800,/Table/52/1/900)
I170402 06:14:54.969699 24919 ccl/storageccl/import.go:57  [n1,s1,r19/1:/Table/52/1/{500-600},@c421536600] import [/Table/52/1/500,/Table/52/1/600)
I170402 06:14:54.970839 24768 ccl/storageccl/import.go:57  [n1,s1,r24/1:/{Table/52/1/9…-Max},@c421903200] import [/Table/52/1/900,/Table/52/2)
I170402 06:14:54.971255 24955 ccl/storageccl/import.go:57  [n1,s1,r26/1:/Table/52/1{-/100},@c423602600] import [/Table/52/1,/Table/52/1/100)
I170402 06:14:54.973236 24932 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader025841622"
I170402 06:14:54.974087 24878 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader776794941"
I170402 06:14:54.974110 24919 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader672579960"
I170402 06:14:54.974282 24768 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader204982391"
I170402 06:14:54.974754 24955 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader205355882"
I170402 06:14:55.246111 24932 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader025841622"
I170402 06:14:55.246896 24853 ccl/storageccl/import.go:57  [n1,s1,r25/1:/Table/52/1/{700-800},@c421ae1200] import [/Table/52/1/700,/Table/52/1/800)
I170402 06:14:55.247188 24853 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader120204737"
I170402 06:14:55.365064 24955 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader205355882"
I170402 06:14:55.365786 24962 ccl/storageccl/import.go:57  [n1,s1,r23/1:/Table/52/1/{100-200},@c4234c5b00] import [/Table/52/1/100,/Table/52/1/200)
I170402 06:14:55.366053 24962 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader291808300"
I170402 06:14:55.373716 24878 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader776794941"
I170402 06:14:55.373730 24768 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader204982391"
I170402 06:14:55.374572 24937 ccl/storageccl/import.go:57  [n1,s1,r22/1:/Table/52/1/{400-500},@c421c6cf00] import [/Table/52/1/400,/Table/52/1/500)
I170402 06:14:55.374665 24854 ccl/storageccl/import.go:57  [n1,s1,r27/1:/Table/52/1/{600-700},@c423711800] import [/Table/52/1/600,/Table/52/1/700)
I170402 06:14:55.374806 24937 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader912490395"
I170402 06:14:55.374936 24854 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader113019966"
I170402 06:14:55.403971 24919 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader672579960"
I170402 06:14:55.416518 24963 ccl/storageccl/import.go:57  [n1,s1,r28/1:/Table/52/1/{300-400},@c4219b4000] import [/Table/52/1/300,/Table/52/1/400)
I170402 06:14:55.416718 24963 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader458346117"
I170402 06:14:55.426494 24853 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader120204737"
I170402 06:14:55.473173 24983 ccl/storageccl/import.go:57  [n2,s2,r23/2:/Table/52/1/{100-200},@c421369200] import [/Table/52/1/100,/Table/52/1/200)
I170402 06:14:55.473386 24983 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader658613280"
I170402 06:14:55.638757 24962 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader291808300"
I170402 06:14:55.639434 25024 ccl/storageccl/import.go:57  [n2,s2,r22/2:/Table/52/1/{400-500},@c42152fb00] import [/Table/52/1/400,/Table/52/1/500)
I170402 06:14:55.644891 25024 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader600894719"
I170402 06:14:55.669013 24854 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader113019966"
I170402 06:14:55.669195 24937 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader912490395"
I170402 06:14:55.669667 24984 ccl/storageccl/import.go:57  [n2,s2,r27/2:/Table/52/1/{600-700},@c420ae6000] import [/Table/52/1/600,/Table/52/1/700)
I170402 06:14:55.669776 24985 ccl/storageccl/import.go:57  [n2,s2,r28/2:/Table/52/1/{300-400},@c422a2f800] import [/Table/52/1/300,/Table/52/1/400)
I170402 06:14:55.672624 24984 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader110624338"
I170402 06:14:55.672674 24985 storage/engine/rocksdb.go:374  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader065766793"
I170402 06:14:55.752158 24963 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader458346117"
I170402 06:14:55.795527 24983 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader658613280"
W170402 06:14:55.832017 23480 storage/raft_transport.go:442  [n1] raft transport stream to node 3 failed: rpc error: code = Internal desc = transport is closing
I170402 06:14:55.832061 25114 util/stop/stopper.go:494  quiescing; tasks left:
3      server/node.go:845
W170402 06:14:55.832157 24792 storage/raft_transport.go:442  [n2] raft transport stream to node 3 failed: rpc error: code = Internal desc = transport is closing
W170402 06:14:55.832231 23216 storage/raft_transport.go:442  [n1] raft transport stream to node 2 failed: rpc error: code = Internal desc = transport is closing
W170402 06:14:55.832285 24797 storage/raft_transport.go:442  [n3] raft transport stream to node 2 failed: rpc error: code = Internal desc = transport is closing
W170402 06:14:55.832304 23251 storage/raft_transport.go:442  [n2] raft transport stream to node 1 failed: rpc error: code = Internal desc = transport is closing
W170402 06:14:55.832316 23506 storage/raft_transport.go:442  [n3] raft transport stream to node 1 failed: rpc error: code = Internal desc = transport is closing
I170402 06:14:55.975518 25024 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader600894719"
I170402 06:14:55.976221 25114 util/stop/stopper.go:494  quiescing; tasks left:
2      server/node.go:845
I170402 06:14:56.004876 24985 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader065766793"
I170402 06:14:56.011751 25114 util/stop/stopper.go:494  quiescing; tasks left:
1      server/node.go:845
I170402 06:14:56.043885 24984 storage/engine/rocksdb.go:441  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupAsOfSystemTime161410566/must-be-cleaned-up/RocksDBSstFileReader110624338"
	backup_test.go:156: found "RocksDBSstFileReader065766793" remaining in tempdir
	backup_test.go:156: found "RocksDBSstFileReader110624338" remaining in tempdir
	backup_test.go:156: found "RocksDBSstFileReader600894719" remaining in tempdir
```

I meant to PR https://github.com/cockroachdb/cockroach/pull/14666 sooner, but looking at the logs, I'm not quite convinced it actually fixes this:

```
[08:45:25][Step 2/2] I170406 08:45:12.824362 2071254 storage/engine/rocksdb.go:372  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/backupRestoreTestSetup461518513/must-be-cleaned-up/RocksDBSstFileReader515163430"
[08:45:25][Step 2/2] I170406 08:45:12.835196 2048194 storage/engine/rocksdb.go:439  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/backupRestoreTestSetup461518513/must-be-cleaned-up/RocksDBSstFileReader118904020"
[08:45:25][Step 2/2] I170406 08:45:12.844694 2071211 ccl/storageccl/import.go:57  [n2,s2,r28/2:/Table/52/1/{600-700},@c430402f00] import [/Table/52/1/600,/Table/52/1/700)
[08:45:25][Step 2/2] I170406 08:45:12.845036 2072969 ccl/storageccl/import.go:57  [n2,s2,r24/2:/Table/52/1/{700-800},@c42d3eef00] import [/Table/52/1/700,/Table/52/1/800)
[08:45:25][Step 2/2] I170406 08:45:12.845554 2071211 storage/engine/rocksdb.go:372  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/backupRestoreTestSetup461518513/must-be-cleaned-up/RocksDBSstFileReader985677901"
[08:45:25][Step 2/2] I170406 08:45:12.846240 2072969 storage/engine/rocksdb.go:372  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/backupRestoreTestSetup461518513/must-be-cleaned-up/RocksDBSstFileReader633594184"
[08:45:25][Step 2/2] W170406 08:45:13.194650 1836834 storage/raft_transport.go:442  [n1] raft transport stream to node 2 failed: rpc error: code = Internal desc = transport is closing
[08:45:25][Step 2/2] W170406 08:45:13.194849 1910433 storage/raft_transport.go:442  [n3] raft transport stream to node 2 failed: rpc error: code = Internal desc = transport is closing
[08:45:25][Step 2/2] W170406 08:45:13.194944 1836880 storage/raft_transport.go:442  [n2] raft transport stream to node 1 failed: EOF
[08:45:25][Step 2/2] I170406 08:45:13.195248 2107221 util/stop/stopper.go:494  quiescing; tasks left:
[08:45:25][Step 2/2] 4      server/node.go:845
[08:45:25][Step 2/2] W170406 08:45:13.195501 1837821 storage/raft_transport.go:442  [n3] raft transport stream to node 1 failed: rpc error: code = Internal desc = transport is closing
[08:45:25][Step 2/2] W170406 08:45:13.195550 1837951 storage/raft_transport.go:442  [n1] raft transport stream to node 3 failed: rpc error: code = Internal desc = transport is closing
[08:45:25][Step 2/2] W170406 08:45:13.195616 1910562 storage/raft_transport.go:442  [n2] raft transport stream to node 3 failed: EOF
[08:45:25][Step 2/2] I170406 08:45:13.206079 2071211 storage/engine/rocksdb.go:439  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/backupRestoreTestSetup461518513/must-be-cleaned-up/RocksDBSstFileReader985677901"
[08:45:25][Step 2/2] I170406 08:45:13.206842 2107221 util/stop/stopper.go:494  quiescing; tasks left:
[08:45:25][Step 2/2] 3      server/node.go:845
[08:45:25][Step 2/2] I170406 08:45:13.208040 2071254 storage/engine/rocksdb.go:439  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/backupRestoreTestSetup461518513/must-be-cleaned-up/RocksDBSstFileReader515163430"
[08:45:25][Step 2/2] I170406 08:45:13.208256 2072969 storage/engine/rocksdb.go:439  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/backupRestoreTestSetup461518513/must-be-cleaned-up/RocksDBSstFileReader633594184"
[08:45:25][Step 2/2] I170406 08:45:13.208772 2107221 util/stop/stopper.go:494  quiescing; tasks left:
[08:45:25][Step 2/2] 2      server/node.go:845
[08:45:25][Step 2/2] I170406 08:45:13.208994 2107221 util/stop/stopper.go:494  quiescing; tasks left:
[08:45:25][Step 2/2] 1      server/node.go:845
[08:45:25][Step 2/2] I170406 08:45:13.210873 2071255 storage/engine/rocksdb.go:439  closing rocksdb instance at "/.../backupRestoreTestSetup461518513/must-be-cleaned-up/RocksDBSstFileReader339827747"
[08:45:25][Step 2/2] --- FAIL: TestBackupAsOfSystemTime (9.71s)
[08:45:25][Step 2/2] 	backup_test.go:156: found "RocksDBSstFileReader339827747" remaining in tempdir
[08:45:25][Step 2/2] 	backup_test.go:156: found "RocksDBSstFileReader515163430" remaining in tempdir
[08:45:25][Step 2/2] 	backup_test.go:156: found "RocksDBSstFileReader633594184" remaining in tempdir
[08:45:25][Step 2/2] 	backup_test.go:156: found "RocksDBSstFileReader985677901" remaining in tempdir
```

so it sees 985677901 sitting there after the test, but we clearly printed above `closing rocksdb instance at "...985677901"` so the call to Close() did happen?

Stange.

--- a/pkg/ccl/sqlccl/backup_test.go
+++ b/pkg/ccl/sqlccl/backup_test.go
@@ -148,13 +148,16 @@ func backupRestoreTestSetupWithParams(
 	}
 
 	cleanupFn := func() {
-		items, err := ioutil.ReadDir(temp)
-		if err != nil && !os.IsNotExist(err) {
-			t.Fatal(err)
-		}
-		for _, leftover := range items {
-			t.Errorf("found %q remaining in tempdir", leftover.Name())
-		}
+		testutils.SucceedsSoon(t, func() error {
+			items, err := ioutil.ReadDir(temp)
+			if err != nil && !os.IsNotExist(err) {
+				t.Fatal(err)
+			}
+			for _, leftover := range items {
+				return errors.Errorf("found %q remaining in tempdir", leftover.Name())
+			}
+			return nil
+		})
 		tc.Stopper().Stop()
 		dirCleanupFn()
 	}

